makefiles/Test.mk

```diff
@@ -45,16 +45,20 @@ test-local:
 test-eks:
 	$(eval VSECM_EKS_CONTEXT=$(shell kubectl config get-contexts -o name | grep "arn:aws:eks"))
 	@if [ -z "$(VSECM_EKS_CONTEXT)" ]; then \
-	echo "Error: No EKS context found."; \
-		exit 1; \
+		echo "Warning: test-eks: No EKS context found."; \
```

Comment:
When we are executing EKS-specific stuff, we should have adequate privileges first.